src/runner.ts

```diff
@@ -46,7 +46,8 @@ export async function run(fn: Runner, args: string[], options: DetectOptions = {
   }
   else {
     let agent = await detect({ ...options, cwd }) || await getDefaultAgent()
-    if (agent === 'prompt') {
+    const isHelp = args.includes('-h') || args.includes('--help')
```

Comment:
```suggestion
    const isHelp = args.length === 1 && ['-h', '--help'].includes(args[0])
```

Also I think it's better to directly emit the help here for early return.

Reply:
yes，no one will add '-h' into the end of args, if they really want to see help